pkg/sliexporter/probes/postgresql_test.go

```diff
@@ -29,14 +29,18 @@ func TestPostgreSQL_Probe(t *testing.T) {
 	require.NoError(t, err)
 	defer pool.Purge(res)
 
+	// race condition exists on my machine, therefore sleep to wait for postgres to be ready
+	// there is unfortunately no way to check if postgres is ready using dockertest
+	time.Sleep(3 * time.Second)
```

Comment:
AFAIK you can get a docker client and list the containers via `pool.Client.ListContainers(opts docker.ListContainersOptions)`. Not sure what it exactly it returns though, but I guess there should be something to determine the readiness of the container or postgresql itself.

Also there's a `pool.Retry()` function which will retry the function you pass to it.

Combining those should give a more robust way to wait for it to be ready.